src/Excel.php

```diff
@@ -193,7 +193,13 @@ protected function findTypeByExtension($fileName, string $type = null): string
             throw new NoTypeDetectedException();
         }
 
-        return config('excel.extension_detector.' . strtolower($extension));
+        $extensionDetector = config('excel.extension_detector.' . strtolower($extension));
+
+        if (!$extensionDetector) {
```

Comment:
This would change behaviour for imports. When finding the readerType there is a fallback when the extension couldn't be found. (IOFactory::identify($filePath)), that won't be called anymore now.

Reply:
Yes I didn't see that one. Now I understand what you meant with. 

> Can you throw the NoTypeDetectedException at the places where this method is used for exports.